### PR TITLE
contact section added

### PR DESCRIPTION
**Pull request by HadertS**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -90,11 +90,31 @@ <h2>Example project</h2>
 				</div>
 			</section>
 			<section id='contact'>
-				
+				<h2>Contact Me</h2>
+				<div>
+					<a href='https://github.com/HadertS'target='_blank' rel='noopener noreferrer'>
```
Comment (Steph-ski) [inline index.html]: Error picked up by HTML validator (space needed between S' & target: 

No space between attributes.
[At line 95, column 42]
ub.com/HadertS'target='_blank'

File: styles.css [hunk 2]
```diff
@@ -161,6 +159,25 @@ section {
...
+}
+
+#contact img{
+	object-fit: cover;
+	
+	height: 15rem;
+	width: 15rem;
+}
+
 @media screen and (max-width: 845px){
```
Comment (Steph-ski) [inline styles.css:181]: Are the media query tags indented to differentiate from the code above?

**Review by Steph-ski — COMMENTED** [review]
Contact section reviewed. 

One error picked up on validator.

 Checking intentional media query indented code.

File: index.html (hunk 1 above)
Comment (HadertS) [reply index.html]: space added

File: styles.css (hunk 2 above)
Comment (HadertS) [reply styles.css:181]: Yes - they're indented for readibility to show they are inside the `@media`tag

**Review by Steph-ski — APPROVED** [review]

**Pull request merged**